src/zadaci/zadatak1/Classroom.java

```diff
@@ -0,0 +1,31 @@
+package zadaci.zadatak1;
+
+import java.util.ArrayList;
+
+public class Classroom {
+
+    private ArrayList<zadaci.zadatak1.Students> studentsList;
+
+    public Classroom(){
+        this.studentsList = new ArrayList<>();
+    }
+
+    public boolean addStudent(final zadaci.zadatak1.Students theStudent){
+        return this.studentsList.add(theStudent);
+    }
+
+    public boolean isPresent(final String name){
+        for (zadaci.zadatak1.Students student : studentsList){
+            if(student.getName() == name){
```

Comment:
compare `Strings` using `equals()` method. This works for `String` for a very specific reason chosen by the creators of Java, but should be avoided for Reference Types (objects) in general.